app/config/AppConfig.scala

```diff
@@ -15,9 +31,9 @@ class AppConfig @Inject()(val runModeConfiguration: Configuration, environment:
   private val contactHost = runModeConfiguration.getString(s"contact-frontend.host").getOrElse("")
```

Comment:
Is is worth getting this to use the loadConfig method that is defined above?

Reply:
The loadConfig method has been removed as part of an AppConfig refactor in another PR (#3 ). In our other services we don't use this function and instead just read the strings defined in ConfigKeys.